File: bench/tasks/05-arrays-each/zig/main.zig
const std = @import("std");

pub fn main() !void {
    const stdout = std.io.getStdOut().writer();
    var scores = [_]i64{ 72, 95, 81, 64 };
    try stdout.print("Students: {}\n", .{scores.len});
    scores[3] = 70;
    for (scores) |score| {
        if (score >= 80) {
            try stdout.print("High score: {}\n", .{score});
        }
    }
}

File: bench/tasks/05-arrays-each/rust/main.rs
fn main() {
    let mut scores = vec![72, 95, 81, 64];
    println!("Students: {}", scores.len());
    scores[3] = 70;
    for score in &scores {
        if *score >= 80 {
            println!("High score: {}", score);
        }
    }
}
